gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfigurationPropertyTest.java

```diff
@@ -9,110 +10,113 @@
-                EntriesCredentialConfiguration.PROXY_ADDRESS_KEY,
-                "proxy-address");
-        config.set(GCS_PROXY_ADDRESS.getKey(),GCS_PROXY_ADDRESS.getDefault());
-        GoogleCloudStorageFileSystemOptions options = optionsBuilder.build();
-
-        assertThat(options.getCloudStorageOptions().getProxyUsername()).isEqualTo("proxy-user");
-        assertThat(options.getCloudStorageOptions().getProxyPassword()).isEqualTo("proxy-pass");
-    }
-
-    @Test(expected = IllegalArgumentException.class)
-    public void testDeprecatedKeys() {
-        GoogleHadoopFileSystemConfigurationProperty<String> GCS_PROXY_ADDRESS =
-                new GoogleHadoopFileSystemConfigurationProperty<>(
-                        EntriesCredentialConfiguration.PROXY_ADDRESS_KEY,
-                        "proxy-address", "fs.gs.proxy.deprecated.address");
-
-        GoogleHadoopFileSystemConfigurationProperty<Integer> GCS_PROXY_USERNAME =
-                new GoogleHadoopFileSystemConfigurationProperty<>(
-                        EntriesCredentialConfiguration.PROXY_USERNAME_KEY,
-                        1234, "fs.gs.proxy.deprecated.user");
-
-        GoogleHadoopFileSystemConfigurationProperty<String> GCS_PROXY_PASSWORD =
-                new GoogleHadoopFileSystemConfigurationProperty<>(
-                        EntriesCredentialConfiguration.PROXY_PASSWORD_KEY,
-                        "proxy-pass", "fs.gs.proxy.deprecated.pass");
-        Configuration config = loadConfig("projectId", "serviceAccount", "privateKeyFile");
-        config.set(GCS_PROXY_ADDRESS.getKey(), GCS_PROXY_ADDRESS.getDefault());
-        config.setInt(GCS_PROXY_USERNAME.getKey(), GCS_PROXY_USERNAME.getDefault());
-        config.set("fs.gs.proxy.deprecated.pass", GCS_PROXY_PASSWORD.getDefault());
-
-        /** Verify that we can read password from config when  used key is deprecated. */
-        String userpass = GCS_PROXY_PASSWORD.getPassword(config);
-        assertThat(userpass).isEqualTo("proxy-pass");
-
-        GoogleCloudStorageFileSystemOptions.Builder optionsBuilder =
-                GoogleHadoopFileSystemConfiguration.getGcsFsOptionsBuilder(config);
-
-        /** Building configuration using deprecated key (in eg. proxy password) should fail. */
-        assertThrows(IllegalArgumentException.class, () -> optionsBuilder.build());
-
-        config.set(GCS_PROXY_PASSWORD.getKey(), GCS_PROXY_PASSWORD.getDefault());
-        GoogleCloudStorageFileSystemOptions options = optionsBuilder.build();
-        assertThat(options.getCloudStorageOptions().getProxyPassword()).isEqualTo("proxy-pass");
-    }
-}
\ No newline at end of file
+            EntriesCredentialConfiguration.PROXY_PASSWORD_KEY,
+            "proxy-pass",
+            "fs.gs.proxy.deprecated.pass");
+    Configuration config = new Configuration();
+    config.set(GCS_PROXY_ADDRESS.getKey(), GCS_PROXY_ADDRESS.getDefault());
+    config.setInt(GCS_PROXY_USERNAME.getKey(), GCS_PROXY_USERNAME.getDefault());
```

Comment:
Should it be `setInt`?

Reply:
Ohh actually the username is 1234, so setInt

Reply:
Is this intentional to use integer type for `GCS_PROXY_USERNAME` property?

Reply:
Kind of. I was wondering if configuration allows to pass integers, but making separate tests for this doesn't make sense, so I made this just a part of test. It seems this could be confusing. Let me know if I should change it to string.

Reply:
Ack, let's left it as it is then.